react-client/styles/_base.scss

```diff
@@ -57,9 +61,10 @@ $DialogDetailPadding: 10px 0px 0px 0px;
 
 
 .focused-targeted-element {
-  z-index: 9999;
-  position: relative;
-  background: #FFFFFF;
+  z-index: 9999 !important;  
+  box-shadow: 999vmax 999vmax  999vmax 9999vmax rgba(0, 0, 0, 0.95);
```

Comment:
Why do we use vmax here?

Reply:
Vmax represents the size that the shadow would cover actually enough nmn  0 0 0 999 vmax pro gn oversize ko n lng just to cover a big space